org.gabmus.razercommander.json

```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "org.gabmus.razercommander",
+    "command": "razercommander",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "razercommander",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--share=network",
+
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.razer",
+
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=~/.config/razercommander:rw",
+        "--filesystem=~/.config:rw",
```

Comment:
Overly broad.

Reply:
I need to create .config/razercommander if it doesnt exist already

Reply:
`--filesystem=~/.config/foo:create` exists for that.

Reply:
cool, didn't know that. will change that if I can keep on using `~/.config/razercommander`